experimental/sql/dialects/iterators.py

```diff
@@ -5,7 +5,7 @@
 from dataclasses import dataclass
 from typing import Any, Union, List
 from xdsl.ir import Block, Region, Operation, SSAValue, ParametrizedAttribute, Data, MLContext, Attribute
-from xdsl.dialects.builtin import StringAttr, ArrayAttr, ArrayOfConstraint, IntegerAttr, IntegerType, TupleType
+from xdsl.dialects.builtin import StringAttr, ArrayAttr, ArrayOfConstraint, IntegerAttr, IntegerType, TupleType, FlatSymbolRefAttr
```

Comment:
Nit, I usually find long imports like this easier to read if put in parens, one per line. My 2 cents.

Reply:
@stellaraccident: Thanks for the suggestion! It has actually sparked a [discussion](https://github.com/xdslproject/xdsl/issues/147) on what convention to use in `xdsl`. Can you recommend a tool that applies the formatting you suggest automatically?